www/front_src/src/Authentication/Openid/models.ts

```diff
@@ -13,40 +13,84 @@ export interface AuthorizationRelationToAPI {
   claim_value: string;
 }
 
+export interface Relations {
+  accessGroup: NamedEntity;
+  claimValue: string;
+}
+
+export interface RelationsToAPI {
+  access_group_id: number;
+  claim_value: string;
+}
+
+export interface EndpointToAPI {
+  custom_endpoint: string | null;
+  type: string;
+}
+
+export interface Endpoint {
+  customEndpoint: string | null;
+  type: string;
+}
+
+export interface RolesMapping {
+  apply_only_first_role: boolean | null;
+  attribute_path: string;
+  authorization_rules: Array<AuthorizationRelationToAPI> | null;
```

Comment:
```suggestion
  authorization_rules: Array<AuthorizationRelationToAPI>;
```
API will only received or return an array (empty or filled) but will not managed null (will throw an error on back end part)